frontend/src/pages/home-page.tsx

```diff
@@ -20,9 +20,8 @@ function HomePage() {
   }, []);
   const navigate = useNavigate();
   return (
-    <div className="font-[Poppins] w-full">
-      {/* <img src={bg} alt="" /> */}
-      <div style={{backgroundImage: `url(${bg})`}} className="relative h-[460px] bg-cover mt-[-8px] bg-fixed bg-center" >
+    <div className="font-[Poppins] w-full dark:bg-dark-bg dark:text-dark-text">
+      <div className="relative h-[460px] bg-cover mt-[-8px] bg-fixed bg-center bg-[url('https://i.ibb.co/Pm4ScWL/wanderlust-bg.webp')]">
```

Comment:
Why are we complicating to use dark:bg-dark-bg ? I assume this is related to tailwind.config.js , this seems confusing , why can't we use directly dark:bg-dark ? from index.css, May i know what is the intension behind this workaround, where it could have been normal

Reply:
sir when there are 2 color then it create conflict. And if we want to give opacity it will not take directly. Then I decided to declare color as variable in config file. @krishnaacharyaa  dark:bg-dark-bg it seems to be complicated but it will be effective when we have more custom colors

Reply:
I respectfully disagree with your thoughts @SankalpHaritash21, dark:bg-dark-bg would be more complicated when we have more custom colors this will be all around the place, not a good dev experience, we need to avoid using this

Reply:
@SankalpHaritash21, Okay after thoroughly going through the code, and referring some docs i figured you are trying to dynamically get the colors, thats good, but just one tip, "I got confused till i saw the code myself" what does it mean ? New contributors will be clueless of how this is applied and where is this coming from, so what do you need to do ? 
**Simplify the complexity**, how do you do that ? By adding good naming conventions.

`bg-dark-bg` can we make it to `bg-dark-background` though it is long but it is now much clear, otherwise you get confused with the tailwind's pre determined classes, isn't it . Same like `text-dark-text` dark-text what ? dark-text color ? font ? size ? correct ? , so make it to `text-dark-text-color`

Reply:
Just one thing, please make if the vs code tailwind extension will pick this up ? otherwise it will be bit of a pain

Reply:
Yes @krishnaacharyaa  tailwind intellisence in vs code is detecting these color.
I will change the name and will give them a standard name.

Reply:
sir I will take some time and will try to come up with better solution this time.
@krishnaacharyaa  can I adjust little bit padding in home page or it will be another issue.
And plz let me know is design good. Plz check Images in PR and let me know

Reply:
Yeah you can add 16px padding on the right and top of featured post card :)